### DISCUSSION TITLE
XY chart hide a single query when clicking on query in legend

### DISCUSSION THREAD

**Discussion by iamelec** [grafana/grafana#62648] [Dashboards, Panels & Visualizations]
The new XY chart looks really promising! Great alternative for the old unsupported Plotly plugin.

I have one issue when there are two or more queries. Both queries are shown in the legend, but when I click either of the queries they both get hidden at once. It would be great if you could hide just one query like in the Time Series graph. 

I might be doing something wrong, but I couldn't find a way to fix this.

All queries shown:
![All queries shown](https://user-images.githubusercontent.com/64543135/215981715-33b39e7a-930c-48ad-ab97-6b509d550a2f.PNG)

When clicking one query, they get all hidden:
![Clicking one query](https://user-images.githubusercontent.com/64543135/215981734-a73aa119-1c0c-4ff6-aa3d-4a29bb1f7d86.PNG)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82862. Feel free to continue the discussion around this there. Thank you!
